reflex/style.py

```diff
@@ -48,7 +48,7 @@ def convert(style_dict):
     var_data = None  # Track import/hook data from any Vars in the style dict.
     out = {}
     for key, value in style_dict.items():
-        key = format.to_camel_case(key)
+        key = format.to_camel_case(key, allow_hyphens=True)
```

Comment:
what is this intending to support? this breaks the code that i was working on to support things like `::-webkit-scrollbar` as pseudo elements

Reply:
I needed this for certain `aria-label` props, that were getting converted automatically to`ariaLabel` and messing it up. I'll see if I can remove this